src/workspaces/common/recentlyViewedWorkspacesUtils.js

```diff
@@ -0,0 +1,17 @@
+import _ from 'lodash/fp';
+import { getLocalPref, setLocalPref } from 'src/libs/prefs';
+
+export const recentlyViewedPersistenceId = 'workspaces/recentlyViewed';
+
+export const updateRecentlyViewedWorkspaces = (workspaceId) => {
```

Comment:
@msilva-broad what is the naming convention for non-component files? Should this be `recently-viewed-workspace-utils`?

Reply:
Taken originally from https://github.com/DataBiosphere/terra-ui/blob/8f9e8efa327b064095818226e6ee77d481b5b755/src/components/workspace-utils.js#L109

Reply:
Never mind-- I have moved this file to /state (because it is state-related) and dropped "Utils" from the end.